modules/ROOT/pages/release-notes.adoc

```diff
@@ -24,6 +24,10 @@ Following the Java 17 update, it is now required to specify the following jvm ar
 --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED
 ----
 
+==== Docker image parent update
+
+The `bonita` and `bonitasoft.jfrog.io/docker/bonita-subscription` images now inherit from the official https://hub.docker.com/_/eclipse-temurin[eclipse-temurin] image in its alpine flavor. As a consequence you may have to update your xref:runtime:bonita-docker-installation.adoc#_adding_internal_ca_certificates_to_the_truststore[internal certificates configuration]. It also affects your Self Contained Application images.
```

Comment:
What is the reason that temurin requires certificates? Any external link to address?

Reply:
It is absolutely not a requirement but an opt-in solution.

Reply:
Ok great